onnxruntime/core/mlas/lib/scalar/SgemmKernelScalar.cpp

```diff
@@ -17,6 +17,53 @@ Module Name:
 
 #include "mlasi.h"
 
+#define MLAS_FLOAT32X4(name) \
```

Comment:
this macro may be confusing as the macro of the same name without parameter is already defined differently

Reply:
Thanks for your review. This macro is defined just for this scope and doesn‘t  influence other code if you don't inlcude this cpp file  (My local test is ok).